### PR TITLE
[OT171-6] Firebase Home Log Event 

### PR DESCRIPTION
**Pull request by marianobattaglia**
Log the following events in Firebase console: 
* 'last_news_see_more_pressed': At tapping "Ver más" section of "Ultimas novedades" list. 
* 'testimonies_see_more_pressed': At tapping "Ver más" section of "Testimonios" list.
* 'slider_retrieve_success': If slider GET has been success.
* 'slider_retrieve_error': If slider GET has been failed.
* 'last_news_retrieve_success': If GET lastest news has been success.
* 'last_news_retrieve_error': If GET lastest news has been failed.
* 'testimonios_retrieve_success': If GET testimonies has been success.
* 'testimonies_retrieve_error': If GET testimonies has been failed.

JIRA Ticket: [OT171-6]
https://alkemy-labs.atlassian.net/browse/OT171-73?atlOrigin=eyJpIjoiYzk0YTZlYzc1MTk5NDllZmE2YzI3MzkzZjcyYzMwNjUiLCJwIjoiaiJ9

### CODE REVIEW

**Review by gapoj — COMMENTED** [review]
Nice work!!!! please take out the prints

File: SomosMasApp/SomosMasApp.xcworkspace/xcshareddata/swiftpm/Package.resolved [hunk 1]
```diff
@@ -0,0 +1,113 @@
+{
+  "pins" : [
+    {
```
Comment (gapoj) [inline SomosMasApp/SomosMasApp.xcworkspace/xcshareddata/swiftpm/Package.resolved:3]: why do you use swift package manager if is not the default manager that we used in the app?

**Review by gapoj — CHANGES_REQUESTED** [review]
please use the same package manager that the rest of the app

File: SomosMasApp/SomosMasApp.xcworkspace/xcshareddata/swiftpm/Package.resolved (hunk 1 above)
Comment (marianobattaglia) [reply SomosMasApp/SomosMasApp.xcworkspace/xcshareddata/swiftpm/Package.resolved]: I dont use cocoapods due to errors while I try to do the integration. Anyway, i investigate how to do it, solved the problem and do PR again using the default manager used in the app.

**Review by gapoj — APPROVED** [review]

**Pull request merged**